package.json

```diff
@@ -29,11 +29,11 @@
     "postinstall": "copyfiles -f node_modules/superstruct/lib/index.js src/lib/superstruct"
   },
   "devDependencies": {
-    "@storybook/addon-info": "^3.3.11",
-    "@storybook/react": "^3.3.11",
-    "cross-env": "^5.1.3",
+    "@storybook/addon-info": "^3.3.15",
+    "@storybook/react": "^3.3.15",
+    "cross-env": "^5.2.0",
```

Comment:
Latest version of cross-env is 5.1.4 (https://www.npmjs.com/package/cross-env) - This is throwing an error on `npm i`

Reply:
D'oh! Thanks for debugging